languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/ignore.txt

```diff
@@ -1980,4 +1980,59 @@ OpenNIC #name
 OpenNet #name
 OpenNap #name
 OpenVMS #name
-rum_sülzen #ugs
\ No newline at end of file
+rum_sülzen #ugs
+Breiding #name
+Malone/S #name
+Maloney/S #name
+NetPositive #name
+Salonen #name
+Description #eng
+Application #eng
+Communications #eng
+Technologies #eng
+California #eng
+Construction #eng
+Acquisition #eng
+Manufacturing #eng
+Architecture #eng
+Relationship #eng
+Marketplace #eng
+Excellence #eng
+Authentication #eng
+Communication #eng
+Stevanovic #name
+Sterniotis #name
+Vasanthakumar #name
+Competition #eng
+Opportunity #eng
+Collaboration #eng
+Redemption #eng
+Configuration #eng
+Care-Arbeit #eng
+Impressions #eng
+Neodigital #name
+Christophe #name
+Unterbuchner #name
+Baccalaureate #eng
+Eichenberger #name
+Subscription #eng
+Bündnerland #name
+Continuity #eng
+Cancellation #eng
+Programming #eng
+Identifier #eng
+Availability #eng
+Bersenbrück #name
+Confirmation #eng
+Technology #eng
+Bretschneider #name
+End-to-End #eng
+Anniversary #eng
+Hospitality #eng
+Brandenberger #name
+Allmendinger #name
+Contemporary #eng
+Certificate #eng
+Call-to-Action/S #eng
+Yevgeny #name
+Prigozhin/S #name
```

Comment:
Die deutsche Schreibweise ist https://de.wikipedia.org/wiki/Jewgeni_Wiktorowitsch_Prigoschin

Reply:
Ich denke dennoch, wir sollten den User nicht korrigieren, wenn er sich für diese Schreibweise entscheidet, daher in `ignore.txt`. Oder soll ich es entfernen?

Reply:
Ich denke schon, dass wir den Nutzer korrigieren sollten. Im journalistischen Kontext umso mehr. Wenn wir den Weg einschlagen wollen, müssten wir auch "Vladimir Putin" oder "Alexei Navalny" etc. akzeptieren. Dazu müsstet ihr erstmal eine Grundsatzentscheidung treffen. Ich würde davon abraten, das Thema jetzt aufzumachen.  https://uebersetzungen-mescheder.de/russische-namen-ins-deutsche-uebertragen/

Reply:
Ok, da das wäre dann ein größerer Task. Dann entferne ich es. Danke!